lib/page/resetpassword/reset_password_page.dart

```diff
@@ -1,17 +1,51 @@
+import 'package:another_flushbar/flushbar.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:survey/constants.dart';
+import 'package:survey/di/di.dart';
 import 'package:survey/gen/assets.gen.dart';
+import 'package:survey/gen/colors.gen.dart';
+import 'package:survey/page/resetpassword/reset_password_state.dart';
+import 'package:survey/page/resetpassword/reset_password_view_model.dart';
 import 'package:survey/resource/dimens.dart';
+import 'package:survey/usecase/reset_password_use_case.dart';
 import 'package:survey/widget/app_bar_back_button_widget.dart';
+import 'package:survey/widget/loading_indicator_widget.dart';
 import 'package:survey/widget/onboarding_background_widget.dart';
 import 'package:survey/widget/rounded_button_widget.dart';
 import 'package:survey/widget/text_input_widget.dart';
 
-class ResetPasswordPage extends StatelessWidget {
-  const ResetPasswordPage({Key? key}) : super(key: key);
+final resetPasswordViewModelProvider = StateNotifierProvider.autoDispose<
+    ResetPasswordViewModel, ResetPasswordState>((ref) {
+  return ResetPasswordViewModel(getIt.get<ResetPasswordUseCase>());
+});
+
+class ResetPasswordPage extends ConsumerStatefulWidget {
+  const ResetPasswordPage({super.key});
+
+  @override
+  _ResetPasswordPageState createState() => _ResetPasswordPageState();
+}
+
+class _ResetPasswordPageState extends ConsumerState<ResetPasswordPage> {
+  final _emailController = TextEditingController();
 
   @override
   Widget build(BuildContext context) {
+    ref.listen<ResetPasswordState>(resetPasswordViewModelProvider, (
+      ResetPasswordState? previousState,
```

Comment:
previousState has no usage, we can hide it by using `_`?

Reply:
@luongvo Since I did this to every screen I created 😅 Do you mind if I include this fix in the latest PR 🙏🏼 I will refactor it for all screens

Reply:
@chornerman yes, no worries

Reply:
Fixed in https://github.com/chornerman/Survey/pull/108